package.json

```diff
@@ -13,22 +13,24 @@
   },
   "scripts": {
     "build": "npm run prepare-dist",
-    "prepare-dist": "uglifyjs src/props/* src/util/* src/components/* -b -o dist/npm/igniteui-react.js && uglifyjs dist/npm/igniteui-react.js -o dist/npm/igniteui-react.min.js",
+    "prepare-dist": "uglifyjs src/props/* -b -o dist/npm/igniteui-react.js && node -e \"require('concat-files')(['build/build-wrapper-start','dist/npm/igniteui-react.js','build/build-wrapper-end'], 'dist/npm/igniteui-react.js');\" && uglifyjs src/util/* dist/npm/igniteui-react.js -b -o dist/npm/igniteui-react.js && node -e \"require('fs-extra').ensureDirSync('dist/npm/ui');require('fs-extra').copySync('src/components', 'dist/npm/ui');require('fs-extra').ensureDirSync('dist/npm/dist');\" && uglifyjs src/util/* dist/npm/igniteui-react.js src/components/* -b -o dist/npm/dist/igniteui-react.js && uglifyjs dist/npm/dist/igniteui-react.js -o dist/npm/dist/igniteui-react.min.js",
```

Comment:
Code from the react core file is duplicated when building the igniteui-react.js file.